docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md

```diff
@@ -80,53 +75,39 @@ In this step, you use a Sumo-provided Azure Resource Manager (ARM) template to c
-      * StorageAccountName: Name of the storage account where  you are storing logs from Azure Service, that you configured in [Step 1](#step-1-configure-azure-storage-account) above.
+      * StorageAccountName: Name of the storage account where you are storing logs from Azure Service that you configured in [Step 1](#step-1-configure-azure-storage-account) above.
       * StorageAccountResourceGroupName: Name of the resource group of the storage account you configured in [Step 1](#step-1-configure-azure-storage-account) above.
-      * Filter Prefix (Optional): If you want to filter logs from a specific container, enter the following, replacing the variable with your container name: `/blobServices/default/containers/<container_name>/`
+      * StorageAccountRegion: Name of the region of the storage account you configured in [Step 1](#step-1-configure-azure-storage-account) above.
+      * Filter Prefix (Optional): If you want to filter logs from a specific container, enter the following by replacing the variable with your container name: `/blobServices/default/containers/<container_name>/`
 
     :::note
     Resource group names should not consist of an underscore.
     :::
-
-1. Select the check box to **agree to the terms and conditions**, and then click **Purchase**.
-
-    ![Azure_Blob_Storage_Custom_Deployment.png](/img/send-data/Azure_Blob_Storage_Custom_Deployment.png)
-
-1. Verify the deployment was successful by looking at **Notifications** at top right corner of Azure Portal.
+1. Go to the **Review + create** tab, and then click **Create**.<br/><img src={useBaseUrl('/img/send-data/Azure_Blob_Storage_Custom_Deployment.png')} alt="Azure_Blob_Storage_Custom_Deploymente" width="400"/>
+1. Verify that the deployment was successful by looking at **Notifications** at the top right corner of the Azure Portal.
 
     ![notification-success.png](/img/send-data/notification-success.png)
 
-1. (Optional) In the same window, click **Go to resource group** to verify the all resources were successfully created, such as shown in the following example:
-
-    ![Azure_Blob_all-resources.png](/img/send-data/Azure_Blob_all-resources.png)
-
-1. Go to **Storage accounts** and search for **sumobrlogs**, then select **sumobrlogs\<*random-string*\\>**.
-
-    ![storage-accounts.png](/img/send-data/storage-accounts.png)
-
-1. Under **Table Service** do the following:
+1. (Optional) In the same window, click **Go to resource group** to verify that all resources were successfully created, such as shown in the following example: <br/><img src={useBaseUrl('/img/send-data/Azure_Blob_all-resources.png')} alt="Azure_Blob_all-resources" width="800"/>
+1. Go to **Storage accounts** and search for **sumobrlogs**, then select **sumobrlogs\<*random-string*\\>**. <br/><img src={useBaseUrl('/img/send-data/storage-accounts.png')} alt="storage-accounts" width="800"/>
+1. Under **Data Storage** do the following:
```

Comment:
```suggestion
1. In the **Data Storage** option, do the following:
```

Reply:
Data Storage is not an option, but only a blade in the left the section of the azure portal window, so under makes sense.
![Uploading Screenshot 2023-06-12 at 5.13.56 PM.png…]()

Reply:
We are referring to User Interface here, so In the Data Storage is more appropriate. It suggests that the actions or items you mention can be found within a specific menu labeled "Data Storage.